libp2p/errors.nim

```diff
@@ -19,7 +19,8 @@ macro checkFutures*[T](futs: seq[Future[T]], exclude: untyped = []): untyped =
         if res.failed:
           let exc = res.readError()
           # We still don't abort but warn
-          warn "Something went wrong in a future", error=exc.name, msg = exc.msg
+          warn "Something went wrong in a future", error=exc.name
```

Comment:
well, isn't this sort of a soft failure that should never happen? the name is not terribly useful

Reply:
Often happens with EOF exceptions.
If you want details enable traces, simple. Without that CI jobs fail cos log is too big.